test/sol5/kyberNetwork.js

```diff
@@ -173,9 +173,8 @@ contract('KyberNetwork', function(accounts) {
         it("test can't add same proxy twice", async() => {
             await tempNetwork.addKyberProxy(proxy1, {from: admin});
 
-            await expectRevert(
-                tempNetwork.addKyberProxy(proxy1, {from: admin}),
-                "proxy exists"
+            await expectRevert.unspecified(
```

Comment:
if we have valid kyberStorage

We still expect same message. no?

Reply:
`kyberStorage` don't have `mapping(address => bool) internal kyberProxyContracts` so it will fail at` kyberNetwork`
would you like to add `kyberProxyContracts` to `kyberStorage`

Reply:
Ok
got it.